src/components/button/_button.scss

```
@@ -569,6 +569,6 @@ $button-shadow-size: 3px;
 
     .ons-btn,
     a {
-        margin: 0 1rem 1rem 0;
+        padding: 0 1.5rem 1rem 0;
```

```suggestion
        margin: 0 1.5rem 1rem 0;
```
This still needs to be a margin